soroban-sdk/src/storage.rs

```diff
@@ -201,22 +201,22 @@ impl Storage {
             .unwrap_infallible();
     }
 
-    pub(crate) fn bump<K>(
+    pub(crate) fn extend<K>(
```

Comment:
I don't really like this name - I would read this as 'extend entry contents' or something. Should it be `extend_ttl`?

Reply:
Agreed, +1

Reply:
updated